packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/lib/base/common_blocs/push_notifications_bloc.dart

```diff
@@ -0,0 +1,58 @@
+{{> licence.dart }}
+
+import 'package:rx_bloc/rx_bloc.dart';
+import 'package:rxdart/rxdart.dart';
+
+import '../../base/extensions/error_model_extensions.dart';
+import '../../base/models/errors/error_model.dart';
+import '../../lib_router/blocs/router_bloc.dart';
+import '../../lib_router/router.dart';
+import '../models/notification_model.dart';
+part 'push_notifications_bloc.rxb.g.dart';
+
+/// A contract class containing all events of the PushNotificationsBloC.
+abstract class PushNotificationsBlocEvents {
+  /// Event for handling opening push notifications from background
+  /// or from in-app
+  void tapOnEvent(NotificationModel event);
+}
+
+/// A contract class containing all states of the PushNotificationsBloC.
+abstract class PushNotificationsBlocStates {
+  ConnectableStream<void> get onRouting;
```

Comment:
What is the purpose of this state? It is not consumed anywhere.

Reply:
This state is used for handling routing on push notification received

Reply:
But is not listened anywhere. In this case it must be a private instance property of Bloc class.